WcaOnRails/config/locales/zh-CN.yml

```diff
@@ -547,12 +566,6 @@ zh-CN:
         #original_hash: da39a3e
         competition_id: ''
       user:
-        #original_hash: 87e1e6f
-        dob: 按“YYYY-MM-DD"格式输入你的生日。
-        #original_hash: d0bb74a
-        name: >-
-          正确输入你的全名，比如<strong>Baiqiang Dong (董百强)</strong>。别偷懒只输入<strong>Baiqiang
-          (百强)</strong>
```

Comment:
I think these hints should be kept as they can be pretty helpful. The en locale uses them as well.